neuralg/__init__.py

```diff
@@ -16,8 +16,11 @@
     download_models()
 
 
-# Set global safe mode variable to false default
-neuralg_SAFEMODE = False
+# Initalize global safe mode variable and set to false default
+from .utils.SafeMode import SafeMode
+
+neuralg_SafeMode = SafeMode()
+neuralg_SafeMode.set_mode(False)
```

Comment:
Mainly because I was too stupid to get it to work when I had it as a variable, it was not updating globally, still not sure why so solved it with a class wrapper cause I knew that worked ...